api/hub/src/datasources/moderation-report.ts

```diff
@@ -195,6 +195,26 @@ class ModerationReportAPI extends DataSource {
         logger.warn(`pending decision could not be created for contentID ${contentID}`);
         // throw new Error('pending decision could not be created');
       }
+      // send email notification to moderators
+      const text = `There is a new pending request for moderation.
+      Please visit the moderation app (https://akasha.ethereum.world/moderation-app/home)
+      to moderate this content.
+      \nThank you!`;
+      const html = `<p>There is a new pending request for moderation.
+      Please visit the <a href="https://akasha.ethereum.world/moderation-app/home">moderation app</a>
+      to moderate this content.</p>
+      <br>
+      <p>Thank you!</p>`;
+      try {
```

Comment:
this should have a debounce period like 1h, or we end up with sending emails for every report made

Reply:
Actually, this email is only sent on the first report for each content ID (when the pending moderation item is created). All subsequent reports will not trigger it.